tests/headless.js

```diff
@@ -3,10 +3,115 @@
 /* eslint-env node */
 "use strict";
 const port = 5000;
-const testURLs = [
-  `http://localhost:${port}/examples/basic.built.html`,
-  `http://localhost:${port}/examples/basic.html`,
+const respec2htmlTests = [
+  {
+    blockDescription: "Process builds",
+    tests: [
+      {
+        URL: `http://localhost:${port}/examples/basic.built.html`,
+        evalFunction: async exec => {
+          try {
+            await exec.run();
+          } catch (error) {
+            throw error;
+          }
+        },
+        message: `Processes example spec employing default respec-w3c-common.js profile`,
+      },
+      {
+        URL: `http://localhost:${port}/examples/basic.html`,
+        evalFunction: async exec => {
+          try {
```

Comment:
Don't need these try/catch blocks I think..

Reply:
Agreed. Only needed when the tests need to play around with errors and warnings.